docs: Put yarn in front of npm (#1968)
vuejs/vuepress: packages/docs/docs/guide/deploy.md

2. Create a file named `.travis.yml` in the root of your project.

> 3. (Only for npm users) Run `npm install` locally and commit `package-lock.json` as it’s required for `npm ci`.

Reviewer: ```suggestion
3. Run `yarn` or `npm install` locally and commit the generated `lock file`.
```
